Python_hw1/hw1.py

```diff
@@ -0,0 +1,20 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+# *Пример:*
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) |
+
+a = str(input("Введите трехзначное число: "))
+if len(str(a)) < 3 or len(str(a)) > 3:
+  print("Число не трехзначное! Введите число, состоящее из трёх цифр: ")
+  a = str(input())
+if len(str(a)) == 3: 
```

Comment:
Данное условие не нужно, так как в 7 строке исключается иное.
```suggestion
else: 
```